.travis.yml

```diff
@@ -1,27 +1,67 @@
 sudo: required
+
 language: scala
+
 jdk:
-  - oraclejdk8
+- oraclejdk8
+
 addons:
   apt:
-    packages: # Required for docker build
-      - net-tools
-# These directories are cached to S3 at the end of the build
+    packages:
+    - net-tools
+
 cache:
   directories:
-    - $HOME/.ivy2/cache
-    - $HOME/.sbt/boot/
+  - "$HOME/.ivy2/cache"
+  - "$HOME/.sbt/boot/"
+
 before_cache:
-  # Tricks to avoid unnecessary cache updates
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
+
 services:
   - docker
+
 scala:
   - 2.11.8
+
 env:
   - DOCKER_DIR="../docker/"
+
 script:
   - sbt ci
+
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
+before_deploy:
+  - sbt assembly
+  - export JAR_DIR="telemetry-streaming"
+  - export JAR_NAME="telemetry-streaming.jar"
+  - mkdir $JAR_DIR
+  - mv target/scala-2.11/telemetry-streaming-assembly-0.1-SNAPSHOT.jar "$JAR_DIR/$JAR_NAME"
+
+deploy:
+  - provider: s3
+    bucket: telemetry-artifacts-stage
```

Comment:
Can we deploy this to `net-mozaws-data-us-west-2-ops-mavenrepo` similar to what we do for telemetry-batch-view?